scripts/var_props.py

```diff
@@ -926,6 +927,13 @@ def __init__(self, var1_stdname, var1_type, var1_kind, var1_units,
                                                                  var2_units)
             # end if
         # end if
+        if self.__compat:
+            # Check for vertical array flipping (do later)
+            if var1_top or var2_top:
```

Comment:
Don't we have to exlude `var1_top and var2_top`?

Reply:
Not sure I follow?
The assumed convention is that top_at_one=False. So any variable (host or scheme) that has top_at_one=True will get transformed before/after calling the schme.

Reply:
Maybe I am confused. if `top_at_one=True` for both the host model variable and the scheme variable, then there's no need to transform?

Reply:
Ah. I see your point. Good catch.
Fixed.